AntennaTracker/system.cpp

```diff
@@ -76,7 +76,7 @@ void Tracker::init_tracker()
     }
 
     // GPS Initialization
-    gps.init(nullptr, serial_manager);
+    gps.init(serial_manager);
```

Comment:
Tracker has MASK_LOG_GPS, any reason not to set it? I know it is a change in behaviour, but I can't see a reason why Tracker wouldn't want to behave like other vehicles.